### PR TITLE
jami: Update to version 202008121023

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

**Review by Ash258 — CHANGES_REQUESTED** [review]
Disable checkver

image

**Comment by linsui** [comment]
I changed the checkver.

File: bucket/jami.json [hunk 1]
```diff
@@ -17,6 +17,10 @@
             "Jami"
         ]
     ],
+    "checkver": {
+        "url": "https://dl.jami.net/windows/archive/?C=M;O=D",
+        "regex": "jami_x86_64-(\\d+).msi"
```
Comment (Ash258) [inline bucket/jami.json:22]:
```suggestion
        "regex": "jami_x86_64-(\\d+)\\.msi"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Pull request merged**